commands/diagnostics.vegman

```diff
@@ -180,6 +186,30 @@ get_intrusion_service() {
          echo Enabled || echo Disabled
 }
 
+get_intrusion_request_status() {
+  local state
+  state=$(busctl get-property ${INTR_DBUS_SERVICE} "${INTR_REQ_STATUS_DBUS_PATH}" \
+                      ${INTR_REQ_STATUS_DBUS_INTF} ${INT_REQ_STATUS_DBUS_PROPERTY} \
+                      2>/dev/null | awk '{print $NF;}' ||:)
+  if [[ -z "${state}" ]]; then
+    echo "clear request state N/A"
+  elif [[ "${state}" == "\"${PROPERTY_PENDING}\"" ]]; then
+    echo "clear request pending"
+  else
+    echo "no clear request pending"
+  fi
+}
+
+set_intrusion_request_status() {
+  local value=$1
+  busctl set-property ${INTR_DBUS_SERVICE} "${INTR_REQ_STATUS_DBUS_PATH}" \
+                      ${INTR_REQ_STATUS_DBUS_INTF} ${INT_REQ_STATUS_DBUS_PROPERTY} \
+                      "s" "${value}" \
+                      2>/dev/null | awk '{print $NF;}' ||:
+
+  echo "Set intrusion sensor request status to:" "${value}"
```

Comment:
Likely we should post the same message to the audit log.
I suggest to add here something like:
```sh
log_audit "Set intrusion sensor request status to: ${value}"
```

BTW: Why have You split a message string and `${value}`? ;)

Reply:
1) should we log here or intel-ipmi-oem?
2) must've missed bracket auto-completion or something, thanks!

Reply:
I believe it should be in both places.
1-th will log the commands sent via IPMI
2-th via CLI.

Reply:
Right. Added the audit logging here and enabled it in the IPMI audit header (that should log IPMI cmds automatically).